### PR TITLE
Update to Django 3.2

### LINKED ISSUE #1544: Update to django 3.2

### PR DESCRIPTION
**Pull request by DmytroLitvinov**
Closes #1544

- [x] Update links to docs 
- [x] Add `asgi.py` file
- [x] Waiting for resolving https://github.com/typeddjango/django-stubs/issues/624

### CODE REVIEW

**Comment by DmytroLitvinov** [comment]
Hi @sobolevn ,
I am planning to upgrade project to Django@3.2.
Do you have any thoughts about what should be upgraded or on what we should pay attention to?

**Review by sobolevn — COMMENTED** [review]
Great work! You also would need to update `django-stubs` after I will make a new release. ETA is this weekend.

File: {{cookiecutter.project_name}}/docker-compose.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@
 version: "3.6"
 services:
   db:
-    image: "postgres:9.6-alpine"
+    image: "postgres:13-alpine"
```
Comment (sobolevn) [inline {{cookiecutter.project_name}}/docker-compose.yml:13]: Great idea! 😆

Comment (DmytroLitvinov) [reply {{cookiecutter.project_name}}/docker-compose.yml:13]: BTW, PostgreSQL v14 Beta 1 is released already :eyes: 

@sobolevn , do you know why there is a problem with DB? 

> 2021/05/27 09:14:57 Problem with dial: dial tcp: lookup db on 127.0.0.11:53: server misbehaving. Sleeping 1s

Comment (sobolevn) [reply {{cookiecutter.project_name}}/docker-compose.yml:13]: Nope, I am still using `9.6`

**Comment by DmytroLitvinov** [comment]
Good evening @sobolevn ,

Something else needs to upgrade to Django@3.2 except the main block of `django-stubs` new release?

**Comment by sobolevn** [comment]
@DmytroLitvinov everything else looks fine! Thanks a lot for your help! 👍

**Comment by sobolevn** [comment]
@DmytroLitvinov `django-stubs@1.9.0` was just released 🙂

**Comment by DmytroLitvinov** [comment]
Hi Nikita,
I am on vacation right now. I can take a look at MR in 1 week. Does it work
for you?

**Comment by sobolevn** [comment]
Sure! Have a great vacaction 🏖️

**Comment by DmytroLitvinov** [comment]
Morning @sobolevn ,
I have updated PR so you can check it out before merging.

**Review by sobolevn — CHANGES_REQUESTED** [review]
This is amazing! Thanks a lot! 👍 

Just a one thing missing: `django-stubs = "^1.9"`

**Comment by DmytroLitvinov** [comment]
@sobolevn , done :+1:

**Review by sobolevn — APPROVED** [review]
Thanks a lot! 🎉 
Big day!

**Comment by sobolevn** [comment]
I am not sure what are the rules are this year, but I've put `hacktoberfest-accept` label for this contribution to count.
Report back if there are any issues with that! 👍

**Comment by DmytroLitvinov** [comment]
Thank you, @sobolevn :relaxed: 

Could you please fix the typo in the label which you added from `hactoberfest-accepted` to `hacktoberfest-accepted`. Here is a rule for that year:

> A repository/project is considered to be participating in Hacktoberfest if the 'hacktoberfest' topic is present and is accepting public contributions via pull requests. An individual pull request can also be opted-in directly by adding the 'hacktoberfest-accepted' label.

**Comment by sobolevn** [comment]
Changed! Should be fine now! 🎃

**Pull request merged**